python/src/lib.rs

```diff
@@ -203,53 +201,61 @@ fn glide(_py: Python, m: &Bound<PyModule>) -> PyResult<()> {
         match val {
             Value::Nil => Ok(py.None()),
             Value::SimpleString(str) => {
-                let data_bytes = PyBytes::new_bound(py, str.as_bytes());
-                Ok(data_bytes.into_py(py))
+                let data_bytes = PyBytes::new(py, str.as_bytes());
+                Ok(data_bytes.into_pyobject(py).unwrap().into_any().unbind())
             }
-            Value::Okay => Ok("OK".into_py(py)),
-            Value::Int(num) => Ok(num.into_py(py)),
+            Value::Okay => Ok("OK".into_pyobject(py).unwrap().into_any().unbind()),
+            Value::Int(num) => Ok(num.into_pyobject(py).unwrap().into_any().unbind()),
             Value::BulkString(data) => {
-                let data_bytes = PyBytes::new_bound(py, &data);
-                Ok(data_bytes.into_py(py))
+                let data_bytes = PyBytes::new(py, &data);
+                Ok(data_bytes.into_pyobject(py).unwrap().into_any().unbind())
             }
             Value::Array(bulk) => {
-                let elements: Bound<PyList> = PyList::new_bound(py, iter_to_value(py, bulk)?);
-                Ok(elements.into_py(py))
+                let elements: Bound<PyList> = PyList::new(py, iter_to_value(py, bulk)?)?;
+                Ok(elements.into_pyobject(py).unwrap().into_any().unbind())
             }
             Value::Map(map) => {
-                let dict = PyDict::new_bound(py);
+                let dict = PyDict::new(py);
                 for (key, value) in map {
                     dict.set_item(resp_value_to_py(py, key)?, resp_value_to_py(py, value)?)?;
                 }
-                Ok(dict.into_py(py))
+                Ok(dict.into_pyobject(py).unwrap().into_any().unbind())
             }
             Value::Attribute { data, attributes } => {
-                let dict = PyDict::new_bound(py);
+                let dict = PyDict::new(py);
                 let value = resp_value_to_py(py, *data)?;
                 let attributes = resp_value_to_py(py, Value::Map(attributes))?;
                 dict.set_item("value", value)?;
                 dict.set_item("attributes", attributes)?;
-                Ok(dict.into_py(py))
+                Ok(dict.into_pyobject(py).unwrap().into_any().unbind())
```

Comment:
I think it'd be better to use `expect` instead of `unwrap` to provide some additional context for why the panic happens. It'll still provide the backtrace, too.

Applies throughout the file.